rust/psibase/src/serve_http.rs

```diff
@@ -178,14 +178,18 @@ pub fn serve_graphql<Query: async_graphql::ObjectType + 'static>(
                 headers: vec![],
             })
         } else {
-            let request = receive_body(
+            let request_result = receive_body(
                 Some(&request.contentType),
                 request.body.as_ref(),
                 Default::default(),
             )
-            .await
-            .unwrap();
-            let res = schema.execute(request).await;
+            .await;
+
+            if let Err(err) = &request_result {
```

Comment:
Rust's new let-else syntax is good for this pattern. We should gradually start moving code towards it.

Reply:
nice! but since we need the Error in this case how would you use it inside the else block?